ibm/provider.go

```diff
@@ -202,6 +202,7 @@ func Provider() terraform.ResourceProvider {
 			"ibm_is_ssh_key":                       dataSourceIBMISSSHKey(),
 			"ibm_is_subnet":                        dataSourceIBMISSubnet(),
 			"ibm_is_subnets":                       dataSourceIBMISSubnets(),
+			"ibm_is_security_group_rule":           dataSourceIBMISSecurityGroupRule(),
```

Comment:
This datasource is for security group not group_rule

"ibm_is_security_group":           dataSourceIBMISSecurityGroup(),

Change the files name and method names to just refer as ibm_is_security_group